### ISSUE TITLE
Integrate 'XUD Master Password' in simulation tests

### ISSUE THREAD

**Issue by kilrau** [ExchangeUnion/xud#1016]
Continuation of https://github.com/ExchangeUnion/xud/issues/912 / https://github.com/ExchangeUnion/xud/issues:

[Simulations tests to use encryption](https://github.com/ExchangeUnion/xud/pull/1006#issuecomment-499826994)

**Comment by kilrau** [comment]
@sangaman could you explain this again?

**Comment by LePremierHomme** [comment]
@sangaman can you please describe what's the desired outcome here?

**Comment by kilrau** [comment]
Please jump in @sangaman , but I'd say something along the lines:
* create node, set random password, cache seed
* lock, unlock with password
* restore node with key

**Comment by sangaman** [comment]
Yes I think that's about right. We want to turn off `noencrypt` and then use the RPC calls to create a wallet, it would be good to also unlock the wallet if possible.

Restoring a node would be trickier, maybe could be in a separate PR, but for that I think we would want to create a node, send funds to at least one LND instance (and ideally one ERC20 token as well), then delete the node and the lnd data dirs/eth keystore. Then we'd want to use the `RestoreNode` rpc call, pass in the mnemonic we got when creating the wallet, and ensure that our node key matches what it did originally and also that the funds are still in our wallets.
